views/stylesheet.css

```diff
@@ -0,0 +1,59 @@
+.calculator-container {
+  width: 232px;
+  height: 323px;
+  border-radius: 15px;
+  position: absolute;
```

Comment:
The specs require this to be centered in the page - the absolute positioning with top, bottom, left, and right set to 0 will stretch this to fill the page. (Also, given this positioning, the `margin: auto` will be ignored.)